docs/user-guide/cli-install-configure-zosmf-security.md

```diff
@@ -1,21 +1,27 @@
 # Configuring z/OSMF Security
 
-Review the tasks that security administrators must complete to configure z/OSMF security for your Zowe CLI implementation.
+Review the tasks that security administrators must complete to configure z/OSMF security for your implementation of Zowe client-side components. **(Is "Zowe client-side components" correct?)**
```

Comment:
@adam-wolfe please respond

Reply:
Rather than "implementation", I might suggest "installation" which I think is slightly clearer.

Reply:
or "configuration" could work